src/sagas/backend.ts

```diff
@@ -600,6 +630,24 @@ async function request(
   }
 }
 
+function* handleResponseError(resp: Response | null) {
+  if (!resp) {
+    yield call(showWarningMessage, "Couldn't reach our servers. Are you online?");
+    return;
+  }
+
+  let errorMessage: string;
+  switch (resp.status) {
+    case 401:
+      errorMessage = 'Session expired. Please login again.';
+      break;
+    default:
+      errorMessage = `Error ${resp.status}: ${resp.statusText}`;
```

Comment:
Might want to consider removing the `status` since it does not add any value to the end user.

Reply:
I think it should be fine, certainly can use better error messages